### PR TITLE
Improve React Native SDK description

### PR DESCRIPTION
**Pull request by andychow326**
ref DEV-2040

### CODE REVIEW

**Comment by louischan-oursky** [comment]
@louischan-oursky

File: packages/authgear-react-native/src/index.ts [hunk 1]
```diff
@@ -908,6 +967,16 @@ export class ReactNativeContainer {
     }
   }
 
+  /**
+   * Enable biometric authentication for the signed in user.
+   * Platform specific biometric authentication UI will be shown.
+   * You may first enable biometric authentication in Authgear Portal (Your Project \> Authentication \> Biometric \> Enable biometric authentication).
+   * Otherwise, `BiometricDisallowed` error will be thrown.
```
Comment (louischan-oursky) [inline packages/authgear-react-native/src/index.ts:974]: We do not have `BiometricDisallowed` error. Remove this line.

File: packages/authgear-react-native/src/index.ts [hunk 2]
```diff
@@ -945,6 +1014,16 @@ export class ReactNativeContainer {
     await this.storage.setBiometricKeyID(this.name, kid);
   }
 
+  /**
+   * Authenticate with biometric authentication.
+   * Platform specific biometric authentication UI will be shown.
+   * You may first enable biometric authentication in Authgear Portal (Your Project \> Authentication \> Biometric \> Enable biometric authentication).
+   * Otherwise, `BiometricDisallowed` error will be thrown.
```
Comment (louischan-oursky) [inline packages/authgear-react-native/src/index.ts:1021]: We do not have `BiometricDisallowed` error. Remove this line.

File: packages/authgear-react-native/src/ui_implementation.ts [hunk 3]
```diff
@@ -1,19 +1,23 @@
...
+ * OpenAuthorizationURLOptions is options for {@link UIImplementation.openAuthorizationURL}.
+ *
  * @public
  */
 export interface OpenAuthorizationURLOptions {
-  /*
-   * The URL to open.
+  /**
+   * The target authorization URL.
    */
```
Comment (louischan-oursky) [inline packages/authgear-react-native/src/ui_implementation.ts:11]: Change the description to `The URL to be opened by the UIImplementation`.

This documentation is for UIImplementation implementor. So from their point of view, they want to know how to implement a UIImplementation. The documentation should guide them.

File: packages/authgear-react-native/src/ui_implementation.ts [hunk 4]
```diff
@@ -1,19 +1,23 @@
...
   url: string;
-  /*
-   * The URL to detect.
+  /**
+   * The value should be a valid Redirect URI to which the response will be sent after authentication.
+   * You must also add a Redirect URI in Authgear Poral via the Redirect URI section of your Authgear Application.
    */
```
Comment (louischan-oursky) [inline packages/authgear-react-native/src/ui_implementation.ts:16]: Change the description to `The URL to be detected by the UIImplementation. When this URL is detected, the UIImplementation MUST return this URL, and close itself`.

File: packages/authgear-react-native/src/ui_implementation.ts [hunk 5]
```diff
@@ -1,19 +1,23 @@
...
   redirectURI: string;
-  /*
+  /**
    * A flag to some implementations that can share cookies with the device browser.
+   * When it is set to true, cookies will be shared with the device browser.
```
Comment (louischan-oursky) [inline packages/authgear-react-native/src/ui_implementation.ts:20]: Change the description to `A flag to tell the UIImplementation that cookies should be shared with the device browser. This flag is only useful to UIImplementation that can share cookies with the device browser, such as those underlying implementations are based ASWebAuthenticationSession, or CustomTabs`.

**Review by louischan-oursky — CHANGES_REQUESTED** [review]

File: packages/authgear-react-native/src/index.ts (hunk 1 above)
Comment (andychow326) [reply packages/authgear-react-native/src/index.ts:974]: ok

File: packages/authgear-react-native/src/index.ts (hunk 2 above)
Comment (andychow326) [reply packages/authgear-react-native/src/index.ts:1021]: ok

File: packages/authgear-react-native/src/ui_implementation.ts (hunk 3 above)
Comment (andychow326) [reply packages/authgear-react-native/src/ui_implementation.ts:11]: ok

File: packages/authgear-react-native/src/ui_implementation.ts (hunk 4 above)
Comment (andychow326) [reply packages/authgear-react-native/src/ui_implementation.ts:16]: ok

File: packages/authgear-react-native/src/ui_implementation.ts (hunk 5 above)
Comment (andychow326) [reply packages/authgear-react-native/src/ui_implementation.ts:20]: ok

**Comment by andychow326** [comment]
@louischan-oursky Updated! :adore2x:

**Review by louischan-oursky — APPROVED** [review]

**Pull request merged**